client/state/sites/domains/reducer.js

```diff
@@ -19,7 +19,10 @@ import {
 	DOMAIN_CONTACT_INFO_REDACT_SUCCESS,
 	DOMAIN_CONTACT_INFO_REDACT_FAILURE,
 	DOMAIN_MARK_AS_PENDING_MOVE,
-} from 'calypso/state/action-types';
+	SITE_DOMAINS_UPDATE_PRIMARY,
+	SITE_DOMAINS_UPDATE_SUCCESS,
+	DOMAIN_MANAGEMENT_PRIMARY_DOMAIN_SAVE_SUCCESS, DOMAIN_MANAGEMENT_PRIMARY_DOMAIN_UPDATE
```

Comment:
What was going on here? Those `SITE_DOMAINS_*` action types were not in use and they introduced ESLint errors. I'm cleaning them up in #90822.